packages/vscode-extension/src/officeChat/types.ts

```diff
@@ -11,3 +11,8 @@ export interface ICopilotChatOfficeResultMetadata {
 export interface ICopilotChatOfficeResult extends ChatResult {
   readonly metadata?: ICopilotChatOfficeResultMetadata;
 }
+
+export type ProjectMiniData = {
```

Comment:
The name `ProjectMiniData` doesn't make sense. Maybe can change to `OfficeProjectInfo` or other reasonable names.

Reply:
Sure, thanks